pkg/api/resource/status.go

```diff
@@ -165,9 +169,11 @@ var statusCodeMessages = []string{
 	"upstream error",       // StatusAPIGroupParentFailed
 	"upstream termination", // StatusAPIGroupParentKilled
 	"update skipped",       // StatusAPIGroupUpdateSkipped
+
+	"killed (oom)", // StatusDataOOM
```

Comment:
How about “terminated (out of memory)”? Does that fit on the cx get line without widening the table? If it’s too long, maybe “terminated (OOM)”, but I worry some people might not know what OOM is